inrupt/pod-browser
# Fix: encode resource iri before fetching

This PR fixes errors when trying to access subfolders with spaces by encoding the iri before fetching.

### To test:
- Create a folder and a subfolder within it, both with names that contain spaces.
- Verify that you can navigate into the subfolder without errors.
